client/lib/site/index.js

```diff
@@ -86,46 +87,7 @@ Site.prototype.set = function( attributes ) {
  * Update any computed attributes
  */
 Site.prototype.updateComputedAttributes = function() {
-	/**
```

Comment:
Ah :facepalm: — I didn't notice you were referring to `lib/site` — long day. Fixed in da7ae9b.